mesheryctl/internal/cli/root/design/testdata/apply.output.golden

```diff
@@ -1,2 +1,2 @@
-design successfully applied
+design applied
 successfully deployed application myapp
```

Comment:
I'm not sure if these lines also will get hit because of previous changes....

Reply:
From the changes made here https://github.com/meshery/meshery/pull/12932/files looks the goal was to remove duplicate `successfully` word in the outputs (which is logic) and if you look all the tests related to this issue are passing now